force-app/main/default/classes/Gifts.cls

```diff
@@ -39,6 +39,17 @@ public with sharing class Gifts {
     @TestVisible private List<DataImport__c> dataImports;
     @TestVisible private List<DataImport__c> giftsToUpdate = new List<DataImport__c>();
 
+    @TestVisible
+    private PS_IntegrationServiceConfig paymentIntegrationServiceConfig {
```

Comment:
[Non-blocking] Similar comment here, unsure if it makes sense to track the integration service as a member of this class.